test/tools/Modules/HttpListener/HttpListener.psm1

```diff
@@ -134,6 +134,13 @@ Function Start-HTTPListener {
                             $contentType = $queryItems["contenttype"]
                             $output = $queryItems["output"]
                         }
+
+                        # Echo the request as the output.
+                        "echo"
```

Comment:
I'm not suggesting always return json.  "response" takes a contenttype argument you can use.